Sources/PackageGraph/Pubgrub/PubgrubDependencyResolver.swift

```diff
@@ -102,30 +102,30 @@ public struct PubgrubDependencyResolver {
     /// The container provider used to load package containers.
     private let provider: ContainerProvider
 
-    /// Skip updating containers while fetching them.
-    private let skipUpdate: Bool
-
     /// Reference to the package container provider.
     private let packageContainerProvider: PackageContainerProvider
 
     /// Should resolver prefetch the containers.
-    private let isPrefetchingEnabled: Bool
+    private let prefetchingEnabled: Bool
+
+    /// Skip updating containers while fetching them.
+    private let updateEnabled: Bool
```

Comment:
The API design guidelines recommend the old names. https://swift.org/documentation/api-design-guidelines/#boolean-assertions

I think the new names can be mis-interpreted as properties that return a new `DependencyMirrors` with prefetching or update enabled, without reading the documentation.